mmselfsup/models/algorithms/maskfeat.py

```diff
@@ -0,0 +1,162 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import math
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+from ..builder import ALGORITHMS, build_backbone, build_head
+from .base import BaseModel
+
+
+class HOGLayerC(nn.Module):
```

Comment:
Better to move `HOGLayer` to a new python file and rename it to `HOGLayer`, like hog_layer.py, and put it in mmselfsup/models/utils/